packages/react-tinacms-editor/src/state/plugins/Menu/index.tsx

```diff
@@ -28,6 +28,7 @@ interface MenuProps {
   bottom?: boolean
   theme?: any
   sticky?: boolean
+  imageUpload?: () => [Promise<string>]
```

Comment:
I know this is still a draft but I just wanted to point this one out
```suggestion
  imageUpload?: (files: File[]) => [Promise<string>]
```